templates/widgets/relax_banner.html

```diff
@@ -0,0 +1,6 @@
+<div class="container-fluid well widget-item relax blue">
+    <a href="/student/relax">
+        <div>На досуге</div>
+    </a>
+</div>
+
```

Comment:
Delete this empty line, but remember to insert a newline char at the end of a file (you should probably enable this in your editor of choice if you haven't done this yet).